### ISSUE TITLE
About the learning rate

### ISSUE THREAD

**Issue by leitro** [fengxinjie/Transformer-OCR#11]
Hi! From the code below I notice that the learning rate starts from 0, and then gradually **increases** from time to time. 
https://github.com/fengxinjie/Transformer-OCR/blob/e3af0159cdbb7f1b639154339b4a97a120c2bf16/train.py#L114
But, what is the intuition to set the learning rate in an ascending mode? Thank you!

**Comment by gussmith** [comment]
As the authors note, this is the same as in 
[https://nlp.seas.harvard.edu/2018/04/03/attention.html](https://nlp.seas.harvard.edu/2018/04/03/attention.html)

**Comment by leitro** [comment]
> As the authors note, this is the same as in
> https://nlp.seas.harvard.edu/2018/04/03/attention.html

That's what I missed. Cheers man! 
**TL;DR:** Increase the learning rate to warm up in early epochs, then decrease it as normal for the following training steps.
